gravity.sh

```diff
@@ -375,10 +369,22 @@ gravity_hostFormatGravity() {
 	echo "" > "${piholeDir}/${accretionDisc}"
 	gravity_doHostFormat "${piholeDir}/${eventHorizon}" "${piholeDir}/${accretionDisc}"
 	# Copy the file over as /etc/pihole/gravity.list so dnsmasq can use it
-	cp ${piholeDir}/${accretionDisc} ${adList}
+	mv "${piholeDir}/${accretionDisc}" "${adList}"
```

Comment:
When removing conflicts, `mv` was the latest addition. Comment still says copy, however.